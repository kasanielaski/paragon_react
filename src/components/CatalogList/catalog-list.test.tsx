import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/react';

import { CatalogList } from '../index';

const props = {
  handleDetails: () => {}
};

describe('CatalogList component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<CatalogList {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
