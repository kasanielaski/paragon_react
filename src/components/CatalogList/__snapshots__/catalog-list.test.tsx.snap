// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CatalogList component > should match snapshot 1`] = `
<DocumentFragment>
  <ul
    class="sc-gjcoXW chsMsb"
  />
</DocumentFragment>
`;
